### ISSUE TITLE
No use of FlowVisor?

### ISSUE THREAD

**Issue by kopida** [sdnwiselab/sdn-wise-java#13]
Hi dear,

In the main program, there is no use of FlowVisor. This means that the flowVisor layer is not activated in the SDN-WISE network, is it? The controller directly communicates with the Adaptation layer, right?

Many thanks in advance!
Chau

**Comment by SebMilardo** [comment]
Yes, in fact the Flowvisor checks if a node is bounded to a Controller and allows packets to get only to the designed controller and vice versa. In the example network there is just one controller so there is no need to add the flowvisor in the loop.

**Comment by kopida** [comment]
Hi SebMilardo,
Thank you very much for your response.
